### PR TITLE
CARML - AVM module transition complete

### PR DESCRIPTION
**Pull request by matebarabas**
# Description

This PR leaves clear indications that the CARML to AVM module transition is complete and that only the AVM version of the modules will be maintained going forward.

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update to documentation

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [ ] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

**Comment by AlexanderSehr** [comment]
The PR looks good, but given it's impact, I want to have a chance to talk with @eriqua first before merging it. 
@eriqua, the main file to review is pretty much only the readme.md in root, as all other changes are removals, updates to the moved-to-avm.md file
image

and updates to each module's readme.md file, which unfortuantely saw a lot more updates than it should as @matebarabas seems to have a markdown VSCode extension with table formatting enabled
image

Not that it matters much at this point. He shall just not do it in AVM 😄

**Comment by eriqua** [comment]
> The PR looks good, but given it's impact, I want to have a chance to talk with @eriqua first before merging it. @eriqua, the main file to review is pretty much only the readme.md in root, image
> 
> as all other changes are removals, updates to the moved-to-avm.md file image
> 
> and updates to each module's readme.md file, which unfortuantely saw a lot more updates than it should as @matebarabas seems to have a markdown VSCode extension with table formatting enabled image
> 
> Not that it matters much at this point. He shall just not do it in AVM 😄

@matebarabas @AlexanderSehr I believe that before removing all modules we should deal with open PRs and make sure that relevant improvements are ported to AVM.
Not sure how PR diffs will render after removing their target code.
I don't see many opened so it will hopefully be quick

File: modules/aad/domain-service/README.md
Comment (eriqua) [inline modules/aad/domain-service/README.md:1]: What is the purpose of keeping the readmes here? I'd suggest to remove them as well.

**Comment by AlexanderSehr** [comment]
> > The PR looks good, but given it's impact, I want to have a chance to talk with @eriqua first before merging it. @eriqua, the main file to review is pretty much only the readme.md in root, image
> > as all other changes are removals, updates to the moved-to-avm.md file image
> > and updates to each module's readme.md file, which unfortuantely saw a lot more updates than it should as @matebarabas seems to have a markdown VSCode extension with table formatting enabled image
> > Not that it matters much at this point. He shall just not do it in AVM 😄
> 
> @matebarabas @AlexanderSehr I believe that before removing all modules we should deal with open PRs and make sure that relevant improvements are ported to AVM. Not sure how PR diffs will render after removing their target code. I don't see many opened so it will hopefully be quick

I presume they'll simply show conflicts all over as they cannot update a file that doesn't exist

File: modules/aad/domain-service/README.md
Comment (AlexanderSehr) [reply modules/aad/domain-service/README.md:1]: cc: @matebarabas

Comment (eriqua) [reply modules/aad/domain-service/README.md:1]: Discussed offline and agreed on removing readmes

File: modules/aad/domain-service/MOVED-TO-AVM.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<h1 style="color: steelblue;">⚠️ Moved to AVM ⚠️</h1>
```
[the remark this answers is not in the document]
Comment (eriqua) [reply modules/aad/domain-service/MOVED-TO-AVM.md:1]: Discussed offline and agreed to keep the current implementation and move to a single file table later if needed. Resolving.

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**